tests/integration/vlan_test.py

```diff
@@ -90,6 +90,22 @@ def test_add_and_remove_two_vlans_on_same_iface(eth1_up):
     assert not current_state[Interface.KEY]
 
 
+def test_two_vlans_on_eth1_change_mtu(eth1_up):
+    desired_state = statelib.show_only(('eth1',))
+    desired_state[Interface.KEY].extend(TWO_VLANS_STATE[Interface.KEY])
+    for iface in desired_state[Interface.KEY]:
+        iface[Interface.MTU] = 2000
+    libnmstate.apply(desired_state)
+
+    del desired_state[Interface.KEY][1]
+    for iface in desired_state[Interface.KEY]:
+        iface[Interface.MTU] = 2200
+    libnmstate.apply(desired_state)
+
+    current_state = statelib.show_only(('eth1.101',))
```

Comment:
do you mind renaming `current_state` to something more descriptive - e.g. `eth1_vlan_iface_current_state`, or something like that ?

I think `current_state` applies for `libnmstate.show()` scenarios - e.g. full state.

Reply:
Sure, done! Thanks